### DISCUSSION TITLE
在新笔记中插入模板报错，请问怎么解决？

### DISCUSSION THREAD

**Discussion by hlwjd** [windingwind/zotero-better-notes#761] [Q&A]
Template [item]English papers MDFiles Error: SyntaxError: JSON.parse: unexpected end of data at line 1 column 1 of the JSON data
![image](https://github.com/windingwind/zotero-better-notes/assets/43723831/556c6b10-72cc-4fbe-a050-1fa58735a487)

**Comment by windingwind** [comment]
Please provide content of the template
